students/AndrewMiotke/lesson01/activity/squarer/test.py

```diff
@@ -0,0 +1,41 @@
+# test.py
+from squarer import Squarer
+
```

Comment:
It looks like you've commented out certain things here that you would want to pass the tests, is this intentional?

Reply:
Huh, not sure why that happened. It looks like all `=` got replaced with a `#`.  I'm guess that happened due to me playing around editors. Thanks for the catch.